app/templates/custom-elements/remote-screen.html

```diff
@@ -77,6 +83,24 @@
           screenImg.addEventListener("contextmenu", (evt) => {
             evt.preventDefault();
           });
+          let isTouchScreen = false;
+          this.shadowRoot.addEventListener("touchend", () => {
+            isTouchScreen = true;
```

Comment:
Oh, this is clever! I wouldn't have thought of this. It might have some false positives at some point, but we can deal with it then.